FusionIIIT/applications/hr2/views.py

```diff
@@ -1081,6 +1205,206 @@ def file_handle(request):
+        form_id = form_data['form_id']
+        file_id = form_data['file_id']
+        from_user = form_data['from_user']
+        from_designation = form_data['from_designation']
+        username_employee = form_data['username_employee']
+        designation_employee = form_data['designation_employee']
+
+        remark = form_data['remark_id']
+        #change
+
+
+        #database
+        try:
+            ltc_form = LTCform.objects.get(id=form_id)
+        except LTCform.DoesNotExist:
+            return JsonResponse({"error": "LTCform object with the provided ID does not exist"}, status=404)
+        
+
+        ltc_form.save()
+        
+
+        current_owner =  get_current_file_owner(file_id)
+        print(current_owner)
+
+
+        if(action == '0'):
+            if(remark == ""):
+                track_id = forward_file(file_id = file_id, receiver = username_employee, receiver_designation = designation_employee,remarks = f"Forwarded by {current_owner} to {username_employee}", file_extra_JSON = "None")
+            else:
+                track_id = forward_file(file_id = file_id, receiver = username_employee, receiver_designation = designation_employee,remarks = f"Forwarded by {current_owner} to {username_employee}, Reason : {remark}", file_extra_JSON = "None")               
+            print("done1",track_id)
+            messages.success(request, "File forwarded successfully")
+        elif(action == '1'):
+            if(remark == ""):
+                track_id = forward_file(file_id = file_id, receiver = ltc_form.name, receiver_designation = ltc_form.designation, remarks = f"Rejected by {current_owner}", file_extra_JSON = "None")
+            else:
+                track_id = forward_file(file_id = file_id, receiver = ltc_form.name, receiver_designation = ltc_form.designation, remarks = f"Rejected by {current_owner}, Reason : {remark}", file_extra_JSON = "None")
+            print("done2")
+            messages.success(request, "File rejected successfully")
+        else:
+            if(remark == ""):
+                track_id = forward_file(file_id = file_id, receiver = ltc_form.name, receiver_designation = ltc_form.designation, remarks = f"Approved by {current_owner}", file_extra_JSON = "None")
+            else:
+                track_id = forward_file(file_id = file_id, receiver = ltc_form.name, receiver_designation = ltc_form.designation, remarks = f"Approved by {current_owner}, Reason : {remark}", file_extra_JSON = "None")
+            print("done3")
+            messages.success(request, "File approved successfully")
+
+        
+        return HttpResponse("Success")
+    else:
+        # Handle other HTTP methods if needed
+        return HttpResponse("Failure")   
+
+
+def file_handle_appraisal(request):
+    if request.method == 'POST':
+        form_data2 = request.POST
+        form_data=request.POST.get('context')
+        print(request.POST.get('context'))
+        action = form_data2.get('action')
```

Comment:
If it's hard to provide descriptive values to the actions, please provide comments regarding which value maps to which action.